src/Icon/resources/styles/icons.scss

```diff
@@ -0,0 +1,17 @@
+.c-icon {
+  &--up {
+    tranform: rotate(0);
+  }
+
+  &--right {
+    transform: rotate(45deg);
+  }
+
+  &--down {
+    transform: rotate(180deg);
+  }
+
+  &--left {
+    transform: rotate(270deg);
+  }
+}
```

Comment:
Do we need these classes or can we just use tailwind?

e.g. `u-rotate-45`

Reply:
We can, I'm just being verbose/OCD about naming 😅  I'll remove.